README.md

```diff
@@ -266,17 +283,24 @@ Asynchronous concurrent distributed networking framework for PHP
 * [clue/php-socket-react](https://github.com/clue/php-socket-react) - Binding for raw sockets (ext-sockets) in [React](#react)
 * [icicleio/socket](https://github.com/icicleio/socket) - Asynchronous stream socket server and client for [Icicle](#icicle)
 * [icicleio/stream](https://github.com/icicleio/stream) - Coroutine-based asynchronous stream interfaces for [Icicle](#icicle)
+* [kraken-php/ipc](https://github.com/kraken-php/ipc) - Asynchronous Socket listener and client
+* [kraken-php/stream](https://github.com/kraken-php/stream) - Asynchronous Streams
 * [reactphp/socket](https://github.com/reactphp/socket) - Asynchronous socket server
 * [reactphp/socket-client](https://github.com/reactphp/socket-client) - Async connector to open TCP/IP and SSL/TLS based connections
 * [reactphp/stream](https://github.com/reactphp/stream) - Asynchronous OO stream wrapper
 * [WyriHaximus/reactphp-psr7-stream-converter](https://github.com/WyriHaximus/reactphp-psr7-stream-converter) - PSR-7 to [React](#react) stream converter and vice versa
 
+### Supervising
+
+* [kraken-php/supervision](https://github.com/kraken-php/supervision) - Supervision system of [Kraken](#kraken)
```

Comment:
It can be used separate from kraken, but would require additional boilerplate for other projects, since it does not include any of kraken solvers. Supervising will probably become a big thing in a while, since almost none PHP async framework controls its processes / threads enought. However as it is for now, after thinking some, I am willing to resign on this one addition, since it might be too early to introduce this topic.